DbFirstApp/Controllers/BrandsController.cs

```diff
@@ -0,0 +1,119 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using DbFirstApp.Data;
+using DbFirstApp.Models;
+
+namespace OnlineShop.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class BrandsController : ControllerBase
+    {
+        private readonly OnlineShopContext _context;
+
+        public BrandsController(OnlineShopContext context)
+        {
+            _context = context;
+        }
+
+        // GET: api/Brands
+        [HttpGet]
+        public async Task<ActionResult<IEnumerable<Brand>>> GetBrands()
+        {
+          if (_context.Brands == null)
+          {
+              return NotFound();
+          }
+            return await _context.Brands.ToListAsync();
+        }
+
+        // GET: api/Brands/5
+        [HttpGet("{id}")]
```

Comment:
Does it work? use `FromRoute` annotation

Reply:
yes, it is